libmachine/drivers/plugin/localbinary/plugin.go

```diff
@@ -188,17 +189,20 @@ func (lbp *Plugin) execServer() error {
 	lbp.addrCh <- strings.TrimSpace(addr)
 
 	stdOutCh, stopStdoutCh := lbp.AttachStream(outScanner)
-	stdErrCh, stopStderrCh := lbp.AttachStream(errScanner)
 
 	for {
 		select {
 		case out := <-stdOutCh:
-			log.Info(fmt.Sprintf(pluginOutPrefix, lbp.MachineName), out)
-		case err := <-stdErrCh:
-			log.Debug(fmt.Sprintf(pluginErrPrefix, lbp.MachineName), err)
+			data := out
+			if strings.Index(out, log.Prefix) == 0 {
+				cut := strings.Index(out, ":")
+				currentLogLevel = out[len(log.Prefix):cut]
+				data = out[cut+1:]
+			}
+			in := []reflect.Value{reflect.ValueOf(fmt.Sprintf(pluginOutPrefix, lbp.MachineName)), reflect.ValueOf(currentLogLevel), reflect.ValueOf(data)}
+			reflect.ValueOf(log.GetLogger()).MethodByName(currentLogLevel).Call(in)
```

Comment:
Reinjecting driver logs into the main process logs at the right level. Dynamic method calls need reflection, then.